src/encoded/tests/test_permissions.py

```diff
@@ -8,59 +8,142 @@ def submission_center_user_app(testapp, test_submission_center, smaht_gcc_user):
 
 
 @pytest.fixture
-def file(testapp, test_submission_center):
-    res = testapp.post_json('/file_format', {
+def consortium_user_app(testapp, test_consortium, smaht_consortium_user):
+    return remote_user_testapp(testapp.app, smaht_consortium_user['uuid'])
+
+
+@pytest.fixture
+def fastq_format(testapp):
+    return testapp.post_json('/file_format', {
         'file_format': 'fastq',
         'standard_file_extension': 'fastq.gz',
         'other_allowed_extensions': ['fq.gz'],
         'valid_item_types': ["FileSubmitted", "FileReference", "FileProcessed"]
     }, status=201).json['@graph'][0]
+
+
+@pytest.fixture
+def submission_center_file(testapp, fastq_format, test_submission_center):
     item = {
-        'file_format': res['uuid'],
+        'file_format': fastq_format['uuid'],
         'md5sum': '00000000000000000000000000000000',
         'filename': 'my.fastq.gz',
```

Comment:
The intent here is to test data model driven permissions, not anything specific about the file itself. This could actually be tested with other types but I happened to use a file type for a more complex case.